pkg-vsn.sh

```diff
@@ -73,6 +73,12 @@ while [ "$#" -gt 0 ]; do
   esac
 done
 
+# return immediately if version is already set
+if [[ -n "${PKG_VSN:-novalue}" != novalue && "${LONG_VERSION:-novalue}" != 'yes' ]]; then
```

Comment:
Maybe if `PKG_VSN` is already defined we can ignore `--long` as well, since it's fully defined?

Reply:
I'd say it makes sense, but I don't know if there is some other logic relying on existing functionality.